platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -8,16 +8,22 @@ supervisorctl start rsyslogd
 
 supervisorctl start syncd
 
+while true; do
+    if [ -e /var/run/sswsyncd/sswsyncd.socket ]; then
+        break
+    fi
+    sleep 1
+done
+
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
     # Wait until syncd has created the socket for bcmcmd to connect to
```

Comment:
@qiluo-msft, I got your point about the comment (1), but for (2), still need your help to clarify it further.